shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/test/java/org/apache/shardingsphere/mode/manager/cluster/coordinator/lock/DistributeLockContextTest.java

```diff
@@ -42,7 +43,7 @@
 import static org.mockito.Mockito.mock;
 
 public final class DistributeLockContextTest {
-    
+
```

Comment:
Please keep the indentation.

Reply:
ok processed

Reply:
Sorry, there are still some unfinished, please check the file again.
![image](https://user-images.githubusercontent.com/12792261/163302445-0f057554-98b1-4aa1-be98-659d1141ee4f.png)

Reply:
ok, you're looking, thanks